OrbitGl/TimerTrack.cpp

```diff
@@ -107,11 +107,22 @@ void TimerTrack::UpdatePrimitives(uint64_t min_tick, uint64_t max_tick,
 
         bool is_visible_width = normalized_length * canvas->GetWidth() > 1;
         bool is_selected = &text_box == GOrbitApp->selected_text_box();
+        bool need_highlight = GOrbitApp->selected_text_box() &&
+                              timer_info.thread_id() != SamplingProfiler::kAllThreadsFakeTid &&
```

Comment:
Is this needed? kAllThreadsFakeTid is used only for sampling events if I'm not mistaken so we shouldn't have to check for it when drawing timers right?

Reply:
Thanks for pointing out this. I misunderstood the purpose for checking kAllThreadsFakeTid. I thought this is used to filter out some uninterested thread. Thanks for explaining this to me : )